### PR TITLE
MD-3770: Added SSL unit test

### PR DESCRIPTION
**Pull request by prasadns14**
@Agirish, please review

### CODE REVIEW

File: framework/pom.xml [hunk 1]
```diff
@@ -37,6 +37,12 @@
       <artifactId>apache-log4j-extras</artifactId>
       <version>${log4j.version}</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/junit/junit -->
```
Comment (Agirish) [inline framework/pom.xml]: May be lets take the comment off?

File: bin/sanity_tests [hunk 2]
```diff
@@ -0,0 +1,7 @@
+source conf/drillTestConfig.properties
```
Comment (Agirish) [inline bin/sanity_tests]: May be call the file run_jdbc_sanity_tests?

File: framework/pom.xml (hunk 1 above)
Comment (prasadns14) [reply framework/pom.xml]: done

File: bin/sanity_tests (hunk 2 above)
Comment (prasadns14) [reply bin/sanity_tests]: done

File: framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java [hunk 3]
```diff
@@ -156,7 +156,7 @@ public void run() {
       }
       duration = stopwatch;
 
-      if (DrillTestDefaults.JDBC_DRIVER.equals(DrillTestDefaults.Driver.APACHE)) {
+      if (DrillTestDefaults.JDBC_DRIVER.equals("org.apache.drill.jdbc.Driver")) {
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java]: How about we create a constant called APACHE_DRIVER in DrillTestDefaults.java and use that in all comparisons?

**Comment by Agirish** [comment]
+1. Thanks a lot for working on this, Prasad. There is a lot we can do in the future to expand this further.

**Pull request merged**